hw1/task2/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Task2
+{
+    /// <summary>Internal class Program.</summary>
+    internal class Program
+    {
+        /// <summary>Function that called when running application.</summary>
+        /// <param name="args">Command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            switch (args.Length == 0 ? "help" : args[0])
+            {
+                case "encode":
+                    {
+                        int indexOfLastChar;
```

Comment:
```suggestion
                        int indexOfLastChar = 0;
```
Даже если переменная инициализируется значением по умолчанию, код становится читаемее, если явно инициализировать